pvlib/spectrum/mismatch.py

```diff
@@ -317,6 +310,14 @@ def spectral_factor_firstsolar(pw, airmass_absolute, module_type=None,
         modules with very similar quantum efficiency should be similar,
         in most cases limiting the need for module specific coefficients.
 
+    min_pw : float, default 0.1
+        minimum atmospheric precipitable water. Any pw value lower than min_pw
+        is set to min_pw to avoid model divergence. [cm]
+
+    max_pw : float, default 8
+        maximum atmospheric precipitable water. Any pw value higher than max_pw
+        is set to NaN to avoid model divergence. [cm]
+
     Returns
     -------
     modifier: array-like
```

Comment:
There's a typo below in the sentence "which is can be multiplied" (I can't edit that part though)